app.rb

```diff
@@ -118,11 +119,11 @@
 # because wget simply fetches the HTML document, this lets us continue to
 # spider the site to generate the contents of everypolitician/viewer-static.
 # See scripts/release.sh (update_viewer_static).
-def docs_redirect(path, page_title)
-  url = URI.join(settings.docs_url, path)
+def redirect(host, path, page_title)
```

Comment:
It seems a little odd to pass a host and path separately when you're only ever going to use them together. Perhaps better to simply require a URI?